contracts/stellar-interchain-token-service/src/tests/execute.rs

```diff
@@ -455,7 +455,13 @@ fn execute_fails_with_invalid_amount() {
 
     let invalid_amount = 0;
     let deployer = Address::generate(&env);
-    let (token_id, _) = setup_its_token(&env, &client, &deployer, invalid_amount);
+    let (token_id, _) = setup_its_token(
+        &env,
+        &client,
+        &deployer,
+        invalid_amount,
+        &Some(Address::generate(&env)),
```

Comment:
think more about what this test is trying to do. it's failing on receiving a transfer msg with zero amount. It's not trying to fail on setting up token. So a minter doesn't need to be passed here. Just use a different initial_supply instead of reusing `invalid_amount` in the wrong place